code/modules/atmospherics/machinery/portable/pump.dm

```diff
@@ -20,8 +16,8 @@
 	var/pressure_limit = 50000
 	///Is the machine on?
 	var/on = FALSE
-	///What direction is the machine pumping (in or out)?
-	var/direction = PUMP_OUT
+	///What direction is the machine pumping (into pump/port or in the tank/area)?
+	var/into_pump_or_port = FALSE
```

Comment:
why drop the defines? the comment here is a bit weird

Reply:
Defines were basically boolean TRUE and FALSE for var/direction, which are now just states of var/into_pump_or_port - whether the pump is sucking into pump/port or not (tank/area).